src/react-components/presence-list.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import styles from "../assets/stylesheets/presence-list.scss";
+import classNames from "classnames";
+import PhoneImage from "../assets/images/mobile_screen_entry.svg";
+import DesktopImage from "../assets/images/desktop_screen_entry.svg";
+import HMDImage from "../assets/images/generic_vr_entry.svg";
```

Comment:
These icons do not look so good on a normal DPI desktop for some reason:

![image](https://user-images.githubusercontent.com/79419/46908228-bc291e00-ced4-11e8-8a35-321fa2e70193.png)

Reply:
ah bc these are svgs. i will bite the bullet and export better PNGs.